### PR TITLE
Merge for Issues #3 and #14

### PR DESCRIPTION
**Pull request by kryptodrex**

### CODE REVIEW

File: trainsim-client/src/components/CheckoutPage.tsx [hunk 1]
```diff
@@ -32,6 +33,33 @@ export default class CheckoutPage extends Component<CheckoutPageProps, CheckoutP
...
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify({
+                    startStopId: itinerary.start.stopId,
+                    endStopId: itinerary.end.stopId,
+                    orderDate: moment().format('YYYY-MM-DD'),
+                    departDate: moment(search.departDate).format('YYYY-MM-DD'),
+                    returnDate: moment(search.returnDate).format('YYYY-MM-DD'),
+                    amountPaid: 10.00,
```
Comment (evanradcliffe) [inline trainsim-client/src/components/CheckoutPage.tsx:49]: This should come from itinerary price * num people? is that out of scope for this PR

Comment (kryptodrex) [reply trainsim-client/src/components/CheckoutPage.tsx]: "amountPaid"? yeah that should come from the itinerary "to" price and (if exists) "from" price (which we can store in Zustand store).

Comment (kryptodrex) [reply trainsim-client/src/components/CheckoutPage.tsx]: Also, just FYI that "travelers" is not # of travelers... It needs to be an array of Traveler json objects

**Comment by kryptodrex** [comment]
@evanradcliffe; This is basically ready. Only thing left is to pull Travelers stored from Zustand store, and add to travelers array on Order object

**Comment by kryptodrex** [comment]
Ok @evanradcliffe I think it's ready now 😄

**Review by evanradcliffe — APPROVED** [review]
👏

**Pull request merged**